### PR TITLE
 Check for Hyper terminal on all operating systems. 

### PR DESCRIPTION
**Pull request by azjezz**
| Q             | A
| ------------- | ---
| Branch?       | 2.8
| Bug fix?      | yes
| New feature?  | no 
| BC breaks?    | no   
| Deprecations? | no 
| Tests pass?   | yes   
| License       | MIT

return true if the terminal is Hyper.is on all operating systems not limited to windows.

### CODE REVIEW

**Comment by azjezz** [comment]
@fabpot 👍

File: src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php [hunk 1]
```diff
@@ -225,14 +225,17 @@ private static function hasColorSupport()
         if (!defined('STDOUT')) {
             return false;
         }
-
+
```
Comment (chalasr) [inline src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php]: extra spaces here, blank line should be unchanged. See fabbot patch in the CI

File: src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php [hunk 2]
```diff
@@ -225,14 +225,17 @@ private static function hasColorSupport()
         if (!defined('STDOUT')) {
             return false;
         }
-
+      
+        if ('Hyper' === getenv('TERM_PROGRAM')) {
+            return true;
+        }
+
```
Comment (chalasr) [inline src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php]: same

**Review by chalasr — APPROVED** [review]
👍 once fabbot happy

**Comment by azjezz** [comment]
@chalasr 👍

**Comment by nicolas-grekas** [comment]
Thank you @azjezz.

**Pull request merged**